prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -27,7 +27,7 @@ class BatchFunction {
             context.logger.info("Batch message: $message")
             val workflowEngine = WorkflowEngine()
             val event = Event.parseQueueMessage(message) as ReceiverEvent
-            if (event.action != Event.Action.BATCH) {
+            if (event.eventAction != Event.EventAction.BATCH) {
```

Comment:
I changed name of Event.Action to Event.EventAction, because the name 'Action' was confusingly overloaded with the database table called ACTION.    This resulted in a zillion little changes throughout the code, as you'll see as you review.

Reply:
Event.EventAction doesn't read well to me. No suggestions, something I'm thinking about.

Reply:
@RickHawesUSDS, can we get rid of EventAction completely, and just use the jooq-generated TaskAction?